embedded-graphics/src/primitives/line_joint.rs

```diff
@@ -200,9 +200,9 @@ impl LineJoint {
                             kind: JointKind::Bevel {
                                 /// Must be counter-clockwise
                                 filler_triangle: Triangle::new(
-                                    first_edge_right.end,
                                     l_intersection,
                                     second_edge_right.start,
+                                    first_edge_right.end,
```

Comment:
I have no clue at all why this helps overdraw.

Reply:
It changes the winding order to be clockwise which I think has some effect on the point-in-triangle algorithm. There must be some numerical inaccuracies in that which cause issues for some edges with certain winding angles. That's my best guess anyway.

Reply:
Didn't I just rotate the triangle? I did not flip 2 points, but shifted everything.

Reply:
Sorry yeah, I've done similar things in the past but swapping the last two points instead of rotating.